.github/workflows/InitDev (Postgres), Tests using Sandbox, Year-Specific and Shared Instance.yml

```diff
@@ -25,19 +25,32 @@ env:
   REPOSITORY_DISPATCH_BRANCH: ${{ github.event.client_payload.branch }}
   EDFI_SUITE_NUMBER: "Suite3"
   msbuild_buildConfiguration: "release"
+  LANG: "en_US.utf8"
```

Comment:
There are a couple of Postman tests that do asserts on date strings, so dates have to be serialized using "en_US.utf8" format (instead of the default ""C.utf8"").